src/styles/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: $calendar-size;
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: all;
+    transition-duration: 0.5s;
+
+    box-sizing: border-box;
+    height: $column-size;
+    width: $column-size;
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    background-color: #eee;
+    border: 1px solid black;
+
+    &:hover {
+      transition-property: transform, background-color;
+      transform: translateY(-20px);
```

Comment:
The 'transform' property should be prefixed with browser-specific prefixes to ensure compatibility with all browsers. For example, you should use '-webkit-transform' for Chrome, Safari, and newer versions of Opera, '-ms-transform' for Internet Explorer, and '-moz-transform' for Firefox.